lib/ansible/module_utils/basic.py

```diff
@@ -616,7 +616,12 @@ def __init__(self, argument_spec, bypass_checks=False, no_log=False,
         self._options_context = list()
         self._tmpdir = None
 
+        self._created_files = []
```

Comment:
Why not use a set instead of a list?

Reply:
I went back and forth on that and ended up just using what other attributes used. I set would have been easier since later I had to remove duplicates from the list just in case.